toolbox/githubctl/main.go

```diff
@@ -303,6 +281,36 @@ func DailyReleaseQualification(baseBranch *string) error {
 	return fmt.Errorf("release qualification failed")
 }
 
+func traverseJobTree(job config.Postsubmit, postsubmitJobs *map[string]struct{}, targetBranch string) {
```

Comment:
Thanks Seb.
Charles, the function definition can be changed to 
func traverseJobTree(job config.Postsubmit, postsubmitJobs map[string]struct{}, targetBranch string) {
notice that postsubmitJobs is declared as a map not a pointer to a map.

Reply:
Done